packages/bot-web-ui/src/stores/dashboard-store.ts

```diff
@@ -138,6 +140,7 @@ export default class DashboardStore implements IDashboardStore {
     strategy_save_type = 'unsaved';
     toast_message = '';
     is_web_socket_intialised = true;
+    is_enabled_modal_chart = false;
```

Comment:
@maryia-matskevich-deriv can we change the variable name to `is_chart_modal_visible`?

and the method to `setChartModalVisibility`?